src/java/voldemort/VoldemortClientShell.java

```diff
@@ -117,7 +117,7 @@ public VoldemortClientShell(ClientConfig clientConfig,
         try {
             factory = new SocketStoreClientFactory(clientConfig);
             client = factory.getStoreClient(storeName);
-            adminClient = new AdminClient(clientConfig);
+            adminClient = new AdminClient(clientConfig.setIdentifierString("admin"));
```

Comment:
What is the impact of de-registering JMX twice?

I don't want to contest whether we need this. I'm just curious to better understand the original issue.